test/recipe/Baked.Test.Recipe.Service.Application/Locales/locale.restext

```diff
@@ -33,14 +33,24 @@ Custom CSS=Custom CSS
 Data Panel=Data Panel
 Data Table=Data Table
 Data_table=Data Table
+DataPanel=Data Panel
```

Comment:
above `Column1` to `Column5` is missing, but I see them in `.baked/locale.en`.

add all missing keys and their values in here, e.g., `Column1=Column 1`